cmake/check_stdint.h.in

```diff
@@ -1,6 +1,41 @@
+/*-*- mode:C; -*- */
+/*
+ * Check: a unit test framework for C
+ *
+ * Copyright (C) 2011 Mateusz Loskot
```

Comment:
Hm, this file was added in 2013. I do not think that Arien or Mateusz have a copyright claim to it. Feel free to add me and yourself instead.

Reply:
Done. Thank you.